app/assets/javascripts/admin/addon/templates/user-index.hbs

```diff
@@ -866,6 +866,42 @@
   </section>
 {{/if}}
 
+{{#if this.currentUser.admin}}
+  <section class="details">
```

Comment:
I think this would be a lot "neater" if you moved this into a component. I realise the rest of the page is probably in the index, but generally we are moving away from these mega route templates. Could be `AdminUserExportsTable` or something, then all the logic from the controller can be moved here too.

Reply:
Good idea, that looks better!